django/db/backends/base/schema.py

```diff
@@ -201,6 +202,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = datetime.now()
```

Comment:
According to the discussion here https://github.com/django/django/pull/4902 for auto_now / auto_now_add no tz should be used for DateField / TimeField (see https://github.com/django/django/pull/4902#discussion_r54287837)